rq_scheduler/scheduler.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, queue_name='default', queue=None, interval=60, connection=Non
         if isinstance(queue_class, string_types):
             self.queue_class_name = queue_class
         self.queue_class = backend_class(self, 'queue_class', override=queue_class)
```

Comment:
Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.